### DISCUSSION TITLE
It's time to say thank you!

### DISCUSSION THREAD

**Discussion by Banana71** [probonopd/MiniDexed#398] [General]
I would like to thank everyone who supports this project. Everyone who asks their questions here, everyone who answers them. Thanks to @probonopd for leading this project brilliantly. Thanks for their enormous expertise to @dcoredump, @arsamus, @BobanSpasic, @rsta2 and many more....
The soul of the DX7 and many great features lives on in MiniDexed. The sound certainly cannot keep up with current devices, but the dynamic capabilities and sound design are unique. I really enjoy making music with the MiniDexed.

**Comment by pik33** [comment]
@Banana71: thank you for these performances you did. I love FM but these presets are far more than I expected after my experiences with Dexed on a PC. 8 instances of DX7+expert programmer=awesome sound.
